pkg/endpoint/endpoint.go

```diff
@@ -63,6 +63,7 @@ import (
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/rest"
 
+	"github.com/cilium/cilium/pkg/metrics"
```

Comment:
Move this import in the Cilium imports group above.

Reply:
done